pydantic/main.py

```diff
@@ -815,6 +817,24 @@ def _get_value(
         else:
             return v
 
+    @classmethod
+    def try_update_forward_refs(cls, **localns: Any) -> None:
+        """
+        Same as update_forward_refs but will not raise exception
+        when forward references are not defined.
+        """
+        if cls.__module__ in sys.modules:
+            globalns = sys.modules[cls.__module__].__dict__.copy()
+        else:
+            globalns = {}
```

Comment:
```suggestion
        module = sys.modules.get(cls.__module__, {'__dict__': {}})
        if module is not None:
            globalns = module.__dict__.copy()
        else:
            globalns = {}
```

Reply:
maybe this should be duplicated (method in `typing.py` or `utils.py`) and used both here and in `update_forward_refs`?

Reply:
Makes sense. I have created function `update_model_forward_refs` at `typing.py`.